File: FileRenamer/Models/About/ReferenceKeyValuePairModel.cpp
#include "pch.h"
#include "ReferenceKeyValuePairModel.h"
#include "ReferenceKeyValuePairModel.g.cpp"

namespace winrt::FileRenamer::implementation
{
	ReferenceKeyValuePairModel::ReferenceKeyValuePairModel() {};

	ReferenceKeyValuePairModel::ReferenceKeyValuePairModel(winrt::hstring const& key, winrt::hstring const& value) :_key{ key }, _value{ value } {};

	winrt::hstring ReferenceKeyValuePairModel::Key()
	{
		return _key;
	}
	void ReferenceKeyValuePairModel::Key(winrt::hstring const& value)
	{
		_key = value;
	}

	winrt::hstring ReferenceKeyValuePairModel::Value()
	{
		return _value;
	}
	void ReferenceKeyValuePairModel::Value(winrt::hstring const& value)
	{
		_value = value;
	}
}

File: FileRenamer/Models/ReferenceKeyValuePairModel.cs
namespace FileRenamer.Models
{
    /// <summary>
    /// 键值对（使用列表模仿字典（因为字典类型无法绑定））数据模型
    /// </summary>
    public class ReferenceKeyValuePairModel
    {
        public string Key { get; set; }

        public string Value { get; set; }
    }
}
